packages/addons/service/pcscd/package.mk

```diff
@@ -0,0 +1,57 @@
+################################################################################
+#      This file is part of LibreELEC - https://libreelec.tv
+#      Copyright (C) 2017-present Team LibreELEC
+#      Copyright (C) 2009-2012 Stephan Raue (stephan@openelec.tv)
+#      Copyright (C) 2011-2011 Gregor Fuis (gujs@openelec.tv)
+#
+#  LibreELEC is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation, either version 2 of the License, or
+#  (at your option) any later version.
+#
+#  LibreELEC is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with LibreELEC.  If not, see <http://www.gnu.org/licenses/>.
+################################################################################
+PKG_NAME="pcscd"
+PKG_VERSION="1.0"
+PKG_REV="100"
+PKG_ARCH="any"
+PKG_LICENSE="GPL"
+PKG_SITE="https://libreelec.tv"
+PKG_URL=""
+PKG_DEPENDS_TARGET="toolchain pcsc-lite libusb ccid"
+PKG_SECTION="service"
+PKG_SHORTDESC="Middleware to access a smart card using SCard API (PC/SC)"
+PKG_LONGDESC="Middleware to access a smart card using SCard API (PC/SC)"
+
```

Comment:
we can add PKG_TOOLCHAIN="manual" and drop make_target and makeinstall_target

Reply:
@InuSasha is this working for every package ?

Reply:
rebased again with the changes